### PR TITLE
Create Agentless monitoring and standardized entity creation

### PR DESCRIPTION
**Pull request by vinrelic**
Deployed earlier today. Please help update the release notes.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by vinrelic** [comment]
This was released earlier today. Please update the release notes.

File: src/content/docs/release-notes/infrastructure-release-notes/cloud-integration-release-notes/Agentless monitoring and standardized entity creation [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Agentless monitoring for EC2 ,Azure VM and standardized entity creation across EC2, azure VMs, and GCP CE is now available. 
+
+## New 
+
+We updated our cloud host monitoring capabilities to help you monitor your Amazon EC2, Azure VM, and GCP CE instances more effectively. You'll now have access to:
+
+* Enhanced visibility into your Amazon EC2, Azure VM infrastructure without the need for complex setup or configuration
+* Simplified set-up that eliminates manual agent installations and configuration 
+* For a cloud resource(EC2, Azure VM, GCP CE) same entity is created when they are instrumented by both agents or as well as in agentless mode(API Polling - AWS/Azure/GCP, Cloudwatch Metric Streams integration, Azure Monitor integration)
```
Comment (bradleycamacho) [inline src/content/docs/release-notes/infrastructure-release-notes/cloud-integration-release-notes/Agentless monitoring and standardized entity creation:15]: Hi @vinrelic, could you try to rewrite this last bullet? I'm a bit confused about what its saying.

**Review by bradleycamacho — CHANGES_REQUESTED** [review]

**Comment by bradleycamacho** [comment]
Hi @vinrelic! I can merge this in as soon as we get that last bullet updated. Feel free to message me on slack for more context.

File: src/content/docs/release-notes/infrastructure-release-notes/cloud-integration-release-notes/Agentless monitoring and standardized entity creation (hunk 1 above)
Comment (vinrelic) [reply src/content/docs/release-notes/infrastructure-release-notes/cloud-integration-release-notes/Agentless monitoring and standardized entity creation:15]: Is it possible to remove the last line? It does not add any value. Can you remove and publish?

**Review by jeff-colucci — APPROVED** [review]

**Pull request merged**